rplugin/python3/floobits/common/exc_fmt.py

```diff
@@ -5,19 +5,19 @@
 import traceback
 
 try:
-    unicode()
+    str()
```

Comment:
String will now always be there. This block and the code below predicating on the existence of str should be able to be removed / flattened-out.